geometry_msgs/Point positionW
geometry_msgs/Point velocityB
geometry_msgs/Point accelB
geometry_msgs/Point rotationE
geometry_msgs/Point angular_vel
float64 joint_angle
float64[6] hydroDynamics
float64[4] internalForce
